.github/workflows/cd_dev.yaml

```diff
@@ -103,13 +95,8 @@ jobs:
         timeout-minutes: 3
         if: ${{ matrix.cluster_type == 'k8' }}
         run: |
-          if [[ ${{ matrix.os }} == 'macos-12' ]]; then
-              export PROXY="--proxy-addr  127.0.0.1"
-            else
-              export PROXY=""
-          fi
```

Comment:
Does the macos-12 k8 not need the PROXY variable anymore? The other removal of PROXY below sets proxy always, so it's not clear if macos uniquely needs a PROXY or if all os combos will use it on some tests?

Reply:
In `CD_DEV` we no longer have `macos-12` as os, only `ubuntu-latest`, so I removed dead code.

in both `CD_DEV` and `CD_DEV_Mac` I removed the condition with `PROXY` and added the actual proxy argument to the start command on `CD_DEV_Mac`.

`--proxy-addr  127.0.0.1` is always needed for macos, and is not needed for ubuntu.